demo/src/assets/resources.js

```diff
@@ -31,7 +31,7 @@ export default {
             style="font-size: 14px; font-weight: 600">{{ name }}</text>
           <text class="component-type"
             x="50" y="3em"
-            style="font-size: 12px; font-style: italic">{{ definition.type }} - id: {{ id }}</text>
+            style="font-size: 12px; font-style: italic">{{ definition.type }} - id: {{ id }} - externalId: {{ externalId }}</text>
```

Comment:
![image](https://github.com/ditrit/leto-modelizer-plugin-core/assets/62500081/af0d2eb0-3a44-4e0b-8f50-a460610458b8)

Some data is out of scope. Maybe put the ids in a new line?

Reply:
Prefere to make:
```
{{Name}}, {{Type}}
id: {{id}}, externalId: {{ExternalId}}
```
Example with DMZ 1:
```
DMZ 1, network
id: network1, externalId: network1
```

Reply:
done